migrator/migrations/frozenschema/v73/groups.go

```diff
@@ -0,0 +1,35 @@
+// Code generated by pg-bindings generator. DO NOT EDIT.
+
+package schema
+
+import (
+	"reflect"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/postgres"
+	"github.com/stackrox/rox/pkg/postgres/walker"
+)
+
+var (
+	// CreateTableGroupsStmt holds the create statement for table `groups`.
+	CreateTableGroupsStmt = &postgres.CreateStmts{
+		GormModel: (*Groups)(nil),
+		Children:  []*postgres.CreateStmts{},
+	}
+
+	// GroupsSchema is the go schema for table `groups`.
+	GroupsSchema = func() *walker.Schema {
+		schema := walker.Walk(reflect.TypeOf((*storage.Group)(nil)), "groups")
+		return schema
+	}()
```

Comment:
The schema seems different. The registration to global table is missing:
```	
GroupsSchema = func() *walker.Schema {
		schema := GetSchemaForTable("groups")
		if schema != nil {
			return schema
		}
		schema = walker.Walk(reflect.TypeOf((*storage.Group)(nil)), "groups")
		RegisterTable(schema, CreateTableGroupsStmt)
		return schema
	}()

```

Reply:
That is on purpose. The dependency is pre-defined and the tables are not to be reused.

Reply:
Okay. What I am worried about is that event if the migrator and its schema is frozen, one can still make changes to the schema/proto which actually will go unhandled. I think we should add a check which scan `pkg/schema/` dir for any changes. If the changes are desirable, the file is added to a whitelist. WDYT?

Reply:
The migration should just have the schema at 3.73. It should not contain future changes to schema. That is what we expect.